_T4Awesome/config.t4ac

```diff
@@ -0,0 +1 @@
+{"PCID":"b79cb3f2-b3aa-411c-a7c3-4813e5da7344","TemplateSearchPaths":[],"Version":"0.0.4","DBConnections":[],"DefaultTemplateLanguage":"C#"}
```

Comment:
As you wrote:

> Please, commit generator here too so other people could contribute to it.

I use T4Awesome tool to generate Tuple classes. That file is its config.